### ISSUE TITLE
Bug relating to nbsp?

### ISSUE THREAD

**Issue by PatentBots** [mozilla/bleach#501]
This is with bleach 3.1:
```
>>> import bleach
>>> bleach.clean(u'foo.&nbsp; bar.')
'foo.&nbsp; bar.'
>>> bleach.linkify(u'foo.&nbsp; bar.')
'foo.\xa0 bar.'
```
Odd that `linkify` would convert `&nbsp;` to `\xa0`.  Maybe related to #143?

Seems like a bug but I'd expect that someone else would have come across this already.

**Comment by g-k** [comment]
Yeah, I think this is related to #143 and the other issues mentioned in https://github.com/mozilla/bleach/issues/453#issuecomment-488314743
